### DISCUSSION TITLE
Am I running `examples/goldencross.jl` correctly?

### DISCUSSION THREAD

**Discussion by g-gundam** [oliviermilla/Lucky.jl#49] [Q&A] [answered]
- I made a fresh clone of oliviermilla/Lucky.jl.
- I ran `julia --project` from the root of the Lucky.jl project.
- I temporarily added MarketData as a dependency since `examples/goldencross.jl` needs it.
- I tried to run it by doing `include("examples/goldencross.jl")`.

It ran without crashing, but I'm not sure if I did it right or if it has a bug, because it looks like it only executed two trades.  It shorted AAPL twice on 1980-12-18, and it never made another trade.  The AAPL data goes to 2013 too.

## The Output
```
[Positions] Data: Position{Cash{USD}, Float64, Date}(Cash{USD}(), 1000.0, Date("1980-12-12"))
[Positions] Data: Position{Stock{:AAPL, USD}, Int64, Date}(Stock{:AAPL, USD}(), 0, Date("1980-12-12"))
[Orders] Data: MarketOrder{Stock{:AAPL, USD}, Int64}(Stock{:AAPL, USD}(), -1)
[Orders] Data: MarketOrder{Stock{:AAPL, USD}, Int64}(Stock{:AAPL, USD}(), -1)
[Positions] Data: Position{Stock{:AAPL, USD}, Int64, Date}(Stock{:AAPL, USD}(), -1, Date("1980-12-18"))
[Fills] Data: Fill{MarketOrder{Stock{:AAPL, USD}, Int64}, Int64, Date}("d257f801-b747-571c-953e-886286c59376", MarketOrder{Stock{:AAPL, USD}, Int64}(Stock{:AAPL, USD}(), -1), 26.62, -1, 0.0, Date("1980-12-18"))
[Positions] Data: Position{Stock{:AAPL, USD}, Int64, Date}(Stock{:AAPL, USD}(), -2, Date("1980-12-18"))
[Fills] Data: Fill{MarketOrder{Stock{:AAPL, USD}, Int64}, Int64, Date}("28155bc3-3fc4-5e82-ad53-e8328ece23b7", MarketOrder{Stock{:AAPL, USD}, Int64}(Stock{:AAPL, USD}(), -1), 26.62, -1, 0.0, Date("1980-12-18"))
```

## Am I doing it wrong?
...or is there a bug in the code?

**Comment by oliviermilla** [comment, marked as answer]
Hey @g-gundam, it's not a bug and it behaves as expected.

It's just a poor logic, only coded to display how strategies should be written. :) 

I should have made a cleaner example though.

**Reply by g-gundam** [reply]
Ok, just making sure.  I think you'll entice more people to look at Lucky.jl if your example makes a lot of imaginary money, though.

**Reply by oliviermilla** [reply]
agreed.
